.build/gcs_upload.yaml

```diff
@@ -19,7 +19,7 @@ options:
 
 steps:
   - id: linux.amd64
-    name: "golang:1.14"
```

Comment:
Was there a particular reason we'd pinned to Go 1.14 here?

Reply:
It was specifically because Go dropped support for cross-compiling for 32-bit Darwin in 1.15. So we decided to compile all of them with the same version, but at our last team sync, we decided to compile all other binaries with 1.16 except for the `darwin.386` one